### PR TITLE
Export mapLoggingT and mapNoLoggingT

### PR DESCRIPTION
**Pull request by mjrussell**
As discussed in #4

### CODE REVIEW

**Review by snoyberg — CHANGES_REQUESTED** [review]
Can you:

* Add Haddocks to the exported functions
* Add a minor version bump to the package.yaml file
* Add a ChangeLog entry
* Add `@since` to the Haddocks for each newly exported function

**Comment by mjrussell** [comment]
Done! I looked at the terminology in the other transformers:
`mapReaderT`: Transform the computation inside a ReaderT.
`mapIdentityT`: Lift a unary operation to the new monad.
`mapExceptT`: Map the unwrapped computation using the given function.

I was debating with how `mapExceptT` described it vs `mapReaderT`. Happy to change it.

File: package.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name:        monad-logger
-version:     0.3.28.5
+version:     0.3.28.6
```
Comment (snoyberg) [inline package.yaml]: The version bump should be to 0.3.29 to conform with the PVP.

**Review by snoyberg — CHANGES_REQUESTED** [review]
I'm fine with the wording, only change is to the version number.

File: package.yaml (hunk 1 above)
Comment (mjrussell) [reply package.yaml]: Ah good call, fixed

**Review by snoyberg — APPROVED** [review]
Thanks!

**Pull request merged**